modules/spacecraftinstruments/rendering/renderableshadowcylinder.cpp

```diff
@@ -39,75 +39,74 @@ namespace {
     };
 
     constexpr openspace::properties::Property::PropertyInfo NumberPointsInfo = {
-        "AmountOfPoints",
+        "NumberOfPoints",
         "Points",
-        "This value determines the number of control points that is used to construct "
-        "the shadow geometry. The higher this number, the more detailed the shadow is, "
-        "but it will have a negative impact on the performance.",
+        "The number of control points to use for constructing the shadow geometry. The "
+        "higher this number, the more detailed the shadow is, but it will have a "
+        "negative impact on the performance.",
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
     constexpr openspace::properties::Property::PropertyInfo ShadowLengthInfo = {
         "ShadowLength",
         "Shadow Length",
-        "This value determines the length of the shadow that is cast by the target "
-        "object. The total distance of the shadow is equal to the distance from the "
-        "target to the Sun multiplied with this value.",
+        "A factor that controls the length of the shadow that is cast by the target "
+        "object. The total length of the shadow is equal to the distance from the "
+        "target to the light source multiplied with this value.",
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
     constexpr openspace::properties::Property::PropertyInfo ShadowColorInfo = {
         "ShadowColor",
         "Shadow Color",
-        "This value determines the color that is used for the shadow cylinder.",
+        "The color to use for the shadow cylinder.",
         openspace::properties::Property::Visibility::User
     };
 
     constexpr openspace::properties::Property::PropertyInfo TerminatorTypeInfo = {
         "TerminatorType",
         "Terminator Type",
-        "This value determines the type of the terminator that is used to calculate the "
-        "shadow eclipse.",
+        "Determines the type of terminator to use for calculating the shadow eclipse "
+        "(UMBRAL or PENUMBRAL).",
```

Comment:
If this is a list of options, I think they should be generated by codegen later on? I.e. no need to specify Umbral and Penumbral here